carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -35,10 +51,31 @@ var subscribe_bsm = () => {
         name: '/bsm_outbound',
         messageType: 'cav_msgs/BSM'
     });
-
     listenerBSM.subscribe(function (message) {
         if (message.core_data != undefined && message.core_data.latitude != undefined && message.core_data.longitude != undefined) {
+            //Vehicle current location
             $("#positionValue").text(message.core_data.latitude + "," + message.core_data.longitude);
+            data.features = [];
+            let feature = createFeature();
+            feature.geometry.coordinates.push(message.core_data.longitude, message.core_data.latitude);
+            data.features.push(feature);
+
+            //Vehicle route
+            if (message.regional != undefined && message.regional.length > 0 && message.regional[0].route_destination_points != undefined
+                && message.regional[0].route_destination_points.length > 0) {
```

Comment:
Is it possible for `message.regional[0].route_destination_points.length` to throw an exception if there is message.regional is an empty list?

Reply:
It will return 0 if list is empty